src/main/java/com/ericsson/ei/frontend/utils/WebControllerUtils.java

```diff
@@ -36,30 +40,42 @@
 @AllArgsConstructor
 public class WebControllerUtils {
 
-    @Value("${ei.frontendServiceHost}")
+    @Value("${ei.frontendServiceHost:#{null}}")
     private String frontendServiceHost;
 
-    @Value("${ei.frontendServicePort}")
+    @Value("${ei.frontendServicePort:#{null}}")
     private int frontendServicePort;
 
-    @Value("${ei.frontendContextPath}")
+    @Value("${ei.frontendContextPath:#{null}}")
     private String frontendContextPath;
 
-    @Value("${ei.useSecureHttpFrontend}")
+    @Value("${ei.useSecureHttpFrontend:#{null}}")
     private boolean useSecureHttpFrontend;
 
-    @Value("${ei.eiffelDocumentationUrls}")
+    @Value("${ei.eiffelDocumentationUrls:#{null}}")
     private String eiffelDocumentationUrls;
 
-    @Value("${spring.application.name}")
-    private String applicationName;
+    @Value("${build.version.name::#{null}}")
+    private String EnterpriseVersionName;
+
+    @Value("${build.version:#{null}}")
+    private String EnterpriseVersion;
 
-    @Value("${build.version}")
     private String version;
 
+    private String applicationName;
+
     @Autowired
     private BackEndInstancesUtils backEndInstancesUtils;
 
+    @PostConstruct
+    public void init() throws IOException {
+        Properties properties = new Properties();
+        properties.load(WebControllerUtils.class.getResourceAsStream("/default-application.properties"));
```

Comment:
have you test executed it with the war file ? : java -jar ei.war , so the classpath works in all situations.

Reply:
Yes.